src/Tribe/Main.php

```diff
@@ -2171,6 +2172,39 @@ public function loadStyle() {
 			}
 		}
 
+		/**
+		 * Before underscore is loaded to the FE we add two scripts on before and one after to prevent underscore from
+		 * taking place on the global namespace if lodash is present.
+		 *
+ 		 * @since TBD
+ 		 *
+ 		 * @param string $tag The <script> tag for the enqueued script.
+		 * @param string $handle The script's registered handle.
+		 * @return string The <script> tag.
+		 */
+		public function prevent_underscore_conflict( $tag, $handle ) {
+
+			if ( is_admin() ) {
+				return $tag;
+			}
+
+			if ( 'underscore' === $handle ) {
+				$path = plugin_dir_path( TRIBE_EVENTS_FILE ) . 'src/resources/js/';
```

Comment:
Take a look at the code from `Tribe__Asset` there are methods there for that. And we should move this method to Common.

Reply:
Cool I'm going to take a look at this one.

Reply:
I don't think there is a method there to do inline JS they are wrappers around `wp_enqueue_script`. What do you think if we add a new method in that class for inline JS.

Also which class do you think it might be the best one to move this logic (most of this PR).

Thanks.

Reply:
I would move it to a new Class `Load_Assets.php` in common.